python/ray/ml/train/data_parallel_trainer.py

```diff
@@ -228,6 +228,7 @@ def __init__(self, train_loop_per_worker, my_backend_config:
         "num_workers",
         "num_cpus_per_worker",
         "num_gpus_per_worker",
+        "resources_per_worker",
         "additional_resources_per_worker",
```

Comment:
Should these other 3 resource keys be removed?

Reply:
Ah yes that's right- we need to remove those.

Reply:
Let's do that in @JiahaoYao's followup!